vllm/v1/core/kv_cache_coordinator.py

```diff
@@ -24,6 +24,7 @@ def __init__(
         use_eagle: bool,
         enable_caching: bool,
         enable_kv_cache_events: bool,
+        dcp_world_size: int,
```

Comment:
do we need to add default value like `dcp_world_size=1` ?

Reply:
<img width="613" height="446" alt="image" src="https://github.com/user-attachments/assets/e048b2ac-6d31-472c-8cd9-4ff21c2dc020" />

dont need now.